pkg/web/static/js/editCounterparty.js

```diff
@@ -0,0 +1,19 @@
+import { countries } from './countriesList.js';
+
+// Use SlimSelect after the counterparty preview partial has been swapped and settled into the DOM.
+document.body.addEventListener('htmx:afterSettle', () => {
+  const countrySelect = new SlimSelect({
+    select: '#country',
+    settings: {
+      contentLocation: document.getElementById('country-list'),
+    },
+  });
+
+  // Get the selected country value.
+  const cpartyCountry = document.getElementById('selected-country');
+  const cpartySelectedCountry = cpartyCountry.value;
+
+  const countriesArray = Object.entries(countries).map(([value, text]) => ({ text, value }));
```

Comment:
Is the countries list referenced by key somewhere else? I'm wondering why we have to create the inverse mapping here every time the DOM loads rather than having that mapping defined in the file.

Reply:
That's a good observation. I should be able to move `countriesArray` to `countries.js` and then import it here and in other files where it's used.